src/js/tracks/video-track.js

```diff
@@ -0,0 +1,54 @@
+import * as VideoTrackEnums from './video-track-enums';
+import Track from './track';
+
+/**
+ * A single video text track as defined in:
+ * @link https://html.spec.whatwg.org/multipage/embedded-content.html#videotrack
+ *
+ * interface VideoTrack {
+ *   readonly attribute DOMString id;
+ *   readonly attribute DOMString kind;
+ *   readonly attribute DOMString label;
+ *   readonly attribute DOMString language;
+ *   attribute boolean selected;
+ * };
+ *
+ * @param {Object=} options Object of option names and values
+ * @class VideoTrack
+ */
+class VideoTrack extends Track {
+  constructor(options = {}) {
+    options.kind = VideoTrackEnums.VideoTrackKind[options.kind] || '';
+    options.trackType = 'video';
+    // on IE8 this will be a document element
+    // for every other browser this will be a normal object
+    let videoTrack = super(options);
+    let selected = false;
+
+    Object.defineProperty(videoTrack, 'selected', {
+      get() { return selected; },
+      set(newSelected) {
+        // an invalid value
+        if(typeof newSelected !== 'boolean' || newSelected === selected) {
+          return;
+        }
+        if(newSelected) {
```

Comment:
Same comment here as for audio track's `enabled` property. The track list seems like the track owner to me so I think it should be responsible for managing cross-track changes like this.